src/item_group.h

```diff
@@ -155,6 +155,7 @@ class Item_modifier
          * Charges to spawn the item with, if this turns out to
          * be negative, the default charges are used.
          */
+        std::pair<double, double> dirt;
```

Comment:
```suggestion
        std::pair<int, int> dirt;
```
Seems double is only needed for the damage probability calculation.

Reply:
was looking for that, thanks.